data/config/__init__.py

```diff
@@ -17,6 +17,17 @@
 DEFAULT_PREFERRED_TEAMS = ["Cubs"]
 DEFAULT_PREFERRED_DIVISIONS = ["NL Central"]
 
+TIME_FORMAT_24H_ALL_PLATFORMS = "%H"
+TIME_FORMAT_12H_NON_WINDOWS   = "%-I"
+TIME_FORMAT_12H_WINDOWS       = "%#I"
+
+TIME_FORMATS_24H = [
+    TIME_FORMAT_24H_ALL_PLATFORMS
+]
+TIME_FORMATS_12H = [
+    TIME_FORMAT_12H_NON_WINDOWS,
+    TIME_FORMAT_12H_WINDOWS
+]
```

Comment:
Any reason to not just do `TIME_FORMAT_12H = if platform.system() == "Windows" then "%#I" else "%-I"`
?

Reply:
Yeah that's a good point

Importing from `data.config` within another data class causes a circular import issue, so I had to put them elsewhere